### PR TITLE
handling filter testcases

### PR DESCRIPTION
**Pull request by mazenmmbadr**

### CODE REVIEW

File: ui_testing/testcases/basic_tests/test05_contacts.py [hunk 1]
```diff
@@ -426,3 +429,233 @@ def test014_edit_approach_overview_button(self):
...
+        data_to_filter_with = self.contacts_api.get_first_record_with_data_in_attribute(attribute='location')
+        self.assertNotEqual(data_to_filter_with, False)
+        self.base_selenium.LOGGER.info('filter with {}'.format(data_to_filter_with))
+        self.contact_page.apply_filter_scenario(filter_element='contact:location', filter_text=data_to_filter_with, field_type='text')
+        table_records = self.contact_page.result_table()
+        counter = 0
+        while counter < (len(table_records) -1):
+            row_data = self.base_selenium.get_row_cells_dict_related_to_header(row=table_records[counter])
+            self.assertEqual(row_data['Location'], data_to_filter_with)
+            counter = counter +1
```
Comment (0xIslamTaha) [inline ui_testing/testcases/basic_tests/test05_contacts.py]: parameterized

Comment (mazenmmbadr) [reply ui_testing/testcases/basic_tests/test05_contacts.py]: what i got from diana, that you wanted each one in a specific testcase for it, not as a parameters

Comment (0xIslamTaha) [reply ui_testing/testcases/basic_tests/test05_contacts.py]: @mazenmmbadr I'm not sure if she got my point clear or not but those cases are identical with different data > parameterized is the best here

Comment (mazenmmbadr) [reply ui_testing/testcases/basic_tests/test05_contacts.py]: but it is implemented in articles and other testcases using the same way, also not all the data are handled the same, so using parameters will be just long test case with multiple if conditions to handle the data that needs special handling

**Pull request merged**